src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -143,6 +239,35 @@ boot_capmgr_mem_alloc(void)
 	cos_meminfo_alloc(capmgr_info, BOOT_MEM_KM_BASE, mem_sz);
 }
 
+/* 
+ * The sinv cap belongs to a different type.
+ * So we can get rid of an extra branch in fast path.
+ */
```

Comment:
Probably place the comment where the `cap` you're referring to is created? 
Or rephrase this to something like: "all `sinv caps` created here are `fault sinv caps`, a different capability type to avoid an extra branch in the invocation fast-path"..